test/browser/fixture.test.js

```diff
@@ -31,6 +27,31 @@ const slowElem = defineCE(
 	}
 );
 
+const asyncElem = defineCE(
+	class extends LitElement {
+		constructor() {
+			super();
+			this.finished = false;
+			this.promise = new Promise((resolve) => resolves.set(this.id, resolve));
```

Comment:
Taking this opportunity to nit unnecessary parentheses, but if this is a pattern I'm fine with it.
```suggestion
			this.promise = new Promise(resolve => resolves.set(this.id, resolve));
```

Reply:
Nah that's just an old unnecessary habit of mine -- thanks.